charts/liquibase-migration/values.yaml

```diff
@@ -1,53 +1,106 @@
-#                                                                                     #
-# # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+# -- Specific migrations to run; each config merged over the defaults to form the full configuration
+# @default -- None by default, one entry here required per migration
+enumerated:
+  - # -- (string) Required name of this specific migration
+    name:
+    # Keys from the defaults may be given here to override them for this specific migration
 
-vault:
-  # Migration credentials only referenced if migration.enabled == true
-  migration:
-    # vault.migration.path -- (string) Required Vault path to secret containing DB credentials
-    path:
-    # vault.migration.dbUsernameKey -- (string) Required key in Vault secret to DB username
-    dbUsernameKey:
-    # vault.migration.DbPasswordKey -- (string) Required key in Vault secret to DB password
-    dbPasswordKey:
+# -- Baseline config merged under each enumerated one to form the full configuration for that particular migration
+# @default -- See sub-keys
+defaults:
 
-migration:
-  # migration.enabled -- (bool) Whether to run a Liquibase migration during sync
+  # -- Whether to enable migrations by default
   enabled: false
-  # migration.imageName -- (string) Required full app image name, without trailing tag
-  # Like "gcr.io/broad-dsp-gcr-public/rawls"
-  imageName:
-  # migration.jarLocation -- (string) Required jar location in app image, expanded by migration.appContainerShell
-  # Like "$(find /rawls -name 'rawls*.jar')"
-  jarLocation:
-  # migration.secretPrefix -- (string) Required prefix of -app-ctmpls, -sqlproxy-ctmpls, -sqlproxy-env secrets
-  # Like "rawls-backend"
-  secretPrefix:
-  # migration.appName -- (string) Name of the application being deployed, for deriving other values
+  
+  # -- (string) Name of the application being deployed
   # @default -- Set to .Release.Name
-  appName: null
-  # migration.labelRef -- (string) Template reference to `include` in resource labels
-  # @default -- Set to "${appName}.labels"
-  labelRef: null
-  # migration.syncWave -- (string) ArgoCD wave to apply these resources in
-  syncWave: "-1"
-  # migration.imageTag -- (string) Tag of the migration.imageName to use
-  # WARNING: App may behave unexpectedly if its database has been migrated on a different version than the app itself
+  k8sAppName: null
+  # -- (string) Template reference to use to obtain resource labels, set to empty to disable
+  # @default -- Set to `${k8sAppName}.labels`
+  k8sLabelRef: null
+  # -- (string) App's Kubernetes service account (selective sync may be necessary if the KSA isn't applied before k8sSyncWave)
+  # @default -- Set to `${k8sAppName}-sa`
+  k8sServiceAccount: null
+  # -- (string) Argo CD wave to apply resources in
+  k8sSyncWave: "-1"
+
+  # -- Image to use for the Cloud SQL Proxy
+  proxyImage: "gcr.io/cloudsql-docker/gce-proxy"
```

Comment:
nit: `sqlproxy` for clarity? We use `proxy` in many Helm charts to refer to the Apache proxy container, and `sqlproxy` for the CloudSQL proxy

Reply:
Good call, I'm going to rename everything in that series of variables, it helps wrangle helm-docs into producing useful output